src/DssSpell.t.sol

```diff
@@ -2042,10 +2038,10 @@ contract DssSpellTest is DSTest, DSMath {
         vat.move(address(this), address(0x0), vat.dai(address(this)));
     }
 
-    function checkUNIV2LPIntegration(
+    function checkUNIV3LPIntegration(
         bytes32 _ilk,
         GemJoinAbstract join,
-        FlipAbstract flip,
+        ClipAbstract flip,
```

Comment:
```suggestion
        ClipAbstract clip,
```

Reply:
done, also changed in the body of the function, and changed `flipMom` for `clipMom`